ui-tests/src/test/java/io/syndesis/qe/steps/other/SupportPageSteps.java

```diff
@@ -0,0 +1,71 @@
+package io.syndesis.qe.steps.other;
+
+import cucumber.api.java.en.When;
+import io.syndesis.qe.pages.SupportPage;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Condition;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+@Slf4j
+public class SupportPageSteps {
+    private SupportPage supportPage = new SupportPage();
+
+
+    @When("^.*downloads diagnostics for all integrations$")
```

Comment:
`downloads` should have trailing question mark like `downloads?` to support both verb variants in every newly added step.